layers/+tools/xclipboard/packages.el

```diff
@@ -0,0 +1,12 @@
+;;; packages.el --- xclipboard layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Google Inc. & Contributors
```

Comment:
This line should be:
```
;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
```

You can add the google one in the `Author` field, you can rename it `Authors`, see https://github.com/syl20bnr/spacemacs/blob//layers/+distributions/spacemacs-base/local/hybrid-mode/hybrid-mode.el#L5-L6